### PR TITLE
Replace and update new datamodel

### PR DESCRIPTION
**Pull request by samleeflang**
- Small updates to the schema to get it in line with the one used in the applications
- Move from the pilot folder to the openDS as this is now the model backing the test environment
  - Split the folder to resemble the individual objects and put the shared models in a separate folder

### CODE REVIEW

File: data-model/shared-models/schema/agent.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "agentName": {
       "type": "string",
       "examples": [
-        "John Smits"
+        "John Smith"
```
Comment (TomDijkema) [inline data-model/shared-models/schema/agent.json:24]: wow

File: data-model/digital-specimen/schema/digital_specimen.json [hunk 2]
```diff
@@ -89,36 +93,56 @@
         "Anthropology",
         "Botany",
         "Astrogeology",
-        "Earth System",
+        "Geology",
```
Comment (TomDijkema) [inline data-model/digital-specimen/schema/digital_specimen.json:96]: Nice catch

File: data-model/digital-specimen/schema/digital_specimen.json [hunk 3]
```diff
@@ -89,36 +93,56 @@
...
         "Microbiology",
         "Palaeontology",
         "Zoology",
-        "Environment",
```
Comment (TomDijkema) [inline data-model/digital-specimen/schema/digital_specimen.json:96]: What happened to Environment? We got a whole category devoted to this on home right now

**Review by TomDijkema — APPROVED** [review]

File: data-model/digital-specimen/schema/digital_specimen.json (hunk 3 above)
Comment (southeo) [reply data-model/digital-specimen/schema/digital_specimen.json:96]: Environment is a topicDomain, not topicDiscipline

File: data-model/digital-specimen/schema/digital_specimen.json [hunk 4]
```diff
@@ -252,19 +276,19 @@
...
       "items": {
-        "$ref": "citations.json"
+        "$ref": "../../shared-models/schema/citations.json"
       }
     },
     "identifiers": {
       "type": "array",
       "items": {
-        "$ref": "identifiers.json"
+        "$ref": "../shared-models/identifiers.json"
```
Comment (southeo) [inline data-model/digital-specimen/schema/digital_specimen.json:291]: is this the right path? the others are `../../shared-models/schema/`

File: data-model/digital-specimen/schema/identifications.json [hunk 5]
```diff
@@ -127,11 +121,11 @@
               "Plants of the World Online"
             ]
           },
-          "dwc:namePublishedInYear": {
+          "dwc:namePublishedInYear":{
```
Comment (southeo) [inline data-model/digital-specimen/schema/identifications.json:124]: restore space

File: data-model/shared-models/schema/agent.json (hunk 1 above)
Comment (southeo) [reply data-model/shared-models/schema/agent.json:24]: is this a joke I'm missing? :sweat_smile:

**Review by southeo — COMMENTED** [review]
looks good, just one small thing

File: data-model/shared-models/schema/agent.json (hunk 1 above)
Comment (samleeflang) [reply data-model/shared-models/schema/agent.json:24]: Might be missing it as well. Maybe it is an inside joke :smile:

File: data-model/digital-specimen/schema/digital_specimen.json (hunk 3 above)
Comment (samleeflang) [reply data-model/digital-specimen/schema/digital_specimen.json:96]: Yes this changed, Tom. The list as it is now should be final

**Review by southeo — APPROVED** [review]
:cool:

File: data-model/shared-models/schema/agent.json (hunk 1 above)
Comment (TomDijkema) [reply data-model/shared-models/schema/agent.json:24]: Just admiring the attention to detail

**Pull request merged**